lms/envs/common.py

```diff
@@ -2856,3 +2856,7 @@
 # Cache time out settings
 # by Comprehensive Theme system
 THEME_CACHE_TIMEOUT = 30 * 60
+
+# API access management
+API_ACCESS_MANAGER_EMAIL = 'api-access@example.com'
+API_ACCESS_FROM_EMAIL = 'api-requests@example.com'
```

Comment:
These values will be overridden by `aws.py`. Move these to `devstack.py`; otherwise, there will be issues when running with devstack.